### PR TITLE
Refetch quote cart before trying to sign in NO

### PR DESCRIPTION
**Pull request by robinandeer**
## What?

Refetch quote cart before trying to sign in NO

Remove quote ids prop from CheckoutPayment component.

## Why?

We sometimes try to sign quotes that are not related to the quote cart.

This happens when we edit quotes during checkout.

Therefore it's better to be safe and refetch the quote cart before trying to sign.

Haven't been able to test since I can't find a working NO SSN...

**Ticket(s): [GRW-1442]**

[GRW-1442]: https://hedvig.atlassian.net/browse/GRW-1442?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* master
  * **PR #1135** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1135" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/web-onboarding/1135?utm_source=stack-comment).

File: src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx [hunk 1]
```diff
@@ -281,6 +281,8 @@ export const CheckoutPayment = ({
     sendDatadogAction('checkout_start')
     try {
       setIsDataLoading(true)
+      const quoteIds = await fetchSelectedQuoteIds()
```
Comment (simonauner) [inline src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx:284]: So the quote ids here might be stale? Before this PR I mean.
Like, there are newer quote ids connected to this user session (quote cart?) ?

**Review by simonauner — APPROVED** [review]
Code-wise LGTM, I just have a question regarding the underlying reason

File: src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx (hunk 1 above)
Comment (robinandeer) [reply src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx:284]: Yes, they apparently good be -- but I haven't dug into exactly why.

We try to refresh the graphql query after editing information but somehow it doesn't propagate all the way.

**Pull request merged**